### ISSUE TITLE
TS Error Received trying to run the PS Invoke-CMApplyDriverPackage Script

### ISSUE THREAD

**Issue by qewade0701** [MSEndpointMgr/ConfigMgr#232]
Please see the attached for the error received during the driver step. Any thoughts would be greatly appreciated.
![DriverMgmtFailure](https://user-images.githubusercontent.com/62809155/82254590-b6998280-9918-11ea-91ec-1ded9d6ff356.png)

**Comment by NickolajA** [comment]
It looks like you've not added PowerShell to your boot image. Could you please verify if it's added?

**Comment by qewade0701** [comment]
Thanks. PowerShell was not in the boot image.

**Comment by qewade0701** [comment]
After adding the PS components the initial was resolved. Thanks. The retry of the TS step resulted in the following. What could I be missing? 
![DriverMgmtFailureII](https://user-images.githubusercontent.com/62809155/82372188-2410e600-99e1-11ea-8dac-99f5bc3767fc.png)

**Comment by NickolajA** [comment]
Check the ApplyDriverPackage.log file for the real error.

**Comment by qewade0701** [comment]
Thanks again for your reply Nickolaj. Really appreciate your responses. The ApplyDriverPackage.log states a successful connection. I wonder if this is my user error :). The pkg I am currently using is a Pilot pkg. The log mentions an operational mode of 'Production'. Is this just a matter of switching the pkg to Production?

![image](https://user-images.githubusercontent.com/62809155/82457446-0f812c00-9a7b-11ea-888e-3c6c00f8cfdd.png)

**Comment by NickolajA** [comment]
Have you added the service account used o the application pool in IIS in ConfigMgr and given it read only permissions?

**Comment by qewade0701** [comment]
Yes. The service acct is in ConfigMgr as a read-only analyst

**Comment by NickolajA** [comment]
Have you created any packages with DAT?

**Comment by qewade0701** [comment]
Yes. The tool has been working fine creating and distributing the initial (Pilot) pkgs.

**Comment by qewade0701** [comment]
I did just move one of those Pilot pkgs to Prod. I haven't tested just yet that outcome.
